config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
 
   resources :posts
+  resources :advertisements
```

Comment:
this creates routes for all CRUD methods. Use `:only => [:index, :show]` etc to limit this.